src/parameterizations/lateral/MOM_self_attr_load.F90

```diff
@@ -27,12 +29,21 @@ module MOM_self_attr_load
   logical :: use_tidal_sal_prev = .false.
     !< If true, read the tidal SAL from the previous iteration of the tides to
     !! facilitate convergence.
-  real    :: sal_scalar_value   !< The constant of proportionality between sea surface height
-                                !! (really it should be bottom pressure) anomalies and bottom
-                                !! geopotential anomalies [nondim].
-  type(sht_CS), allocatable :: sht  !< Spherical harmonic transforms (SHT) control structure
-  integer :: sal_sht_Nd         !< Maximum degree for SHT [nodim]
-  real, allocatable :: Love_Scaling(:) !< Love number for each SHT mode [nodim]
+  logical :: use_bpa = .false.
+    !< If true, use bottom pressure anomaly instead of SSH to calculate SAL.
+  real :: eta_prop
+    !< The partial derivative of eta_sal with the local value of eta [nondim].
+  real :: linear_scaling
+    !< Dimensional coefficients for scalar SAL [nondim or Z T2 L-2 R-1 ~> m Pa-1]
+  type(sht_CS), allocatable :: sht
+    !< Spherical harmonic transforms (SHT) control structure
+  integer :: sal_sht_Nd
+    !< Maximum degree for spherical harmonic transforms [nodim]
+  real, allocatable :: ebot_ref(:,:)
+    !< Reference bottom pressure scaled by Rho_0 and G_Earth[Z ~> m]
+  real, allocatable :: Love_scaling(:)
+    !< Dimensional coefficients for harmonic SAL, which are functions of Love numbers
+    !! [nondim or Z T2 L-2 R-1 ~> m Pa-1]
```

Comment:
Please revise this comment to read:
`!! [nondim] or [Z T2 L-2 R-1 ~> m Pa-1], depending on the value of use_ppa.`